create.py

```diff
@@ -1,31 +1,34 @@
 import os
-from flask import Flask, request, render_template
+
+from flask import Flask, request, render_template,url_for
 from models import *
+from books import *
 from flask import Flask, session, redirect
 from flask_session import Session
-from sqlalchemy import create_engine, desc
+from sqlalchemy import create_engine, desc , or_
 from sqlalchemy.orm import scoped_session, sessionmaker
 import logging
 
 logging.basicConfig(level=logging.DEBUG)
+
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = os.getenv("DATABASE_URL")
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 app.config["SESSION_PERMANENT"] = False
 app.config["SESSION_TYPE"] = "filesystem"
+
 engine = create_engine(os.getenv("DATABASE_URL"))
 
 Session(app)
 db.init_app(app)
 
 @app.route('/')
 def hello_world():
-    return redirect('/register')
-@app.route('/register', methods = ['POST','GET'])
+    return redirect('/registration')
+@app.route('/registration', methods = ['POST','GET'])
 def register():
     db.create_all()
```

Comment:
as said earlier create_all is used to create a table and is not required to be called everytime the /registration route is called.

Reply:
Changed Bro